apps/extension/jest.config.js

```diff
@@ -6,7 +6,7 @@ const { compilerOptions } = require("./tsconfig.json")
 
 module.exports = {
   transformIgnorePatterns: [
-    "/node_modules/(?!@polkadot|@substrate|@azns|@babel/runtime/helpers/esm/|@metamask|url-join|isows|nanoid)",
+    "/node_modules/.pnpm/(?!@polkadot|@substrate|@azns|@babel/runtime/helpers/esm/|@metamask|url-join|isows|nanoid|@talismn|scale-ts)",
```

Comment:
Because of pnpm's [isolated](https://pnpm.io/npmrc#node-linker) `node_modules` structure, jest finds all of the dependencies inside of `node_modules/.pnpm`